core/LSU.cpp

```diff
@@ -925,6 +612,64 @@ namespace olympia
 
     }
 
+    ////////////////////////////////////////////////////////////////////////////////
+    // MMU subroutines
+    ////////////////////////////////////////////////////////////////////////////////
+    // Handle MMU access request
+    void LSU::handleMMULookupReq1_()
+    {
+        const auto stage_id = static_cast<uint32_t>(PipelineStage::MMU_LOOKUP);
+
+        // Check if flushing event occurred just now
+        if (!ldst_pipeline_.isValid(stage_id)) {
+            return;
+        }
+
+        const MemoryAccessInfoPtr & mem_access_info_ptr = ldst_pipeline_[stage_id];
+        ILOG(mem_access_info_ptr);
+
+        bool isAlreadyHIT = (mem_access_info_ptr->getMMUState() == MemoryAccessInfo::MMUState::HIT);
+        bool MMUBypass = isAlreadyHIT;
```

Comment:
Duplicate `bool` variables.  Remove one; make the other `const` and change the variable name to match the rest of the code